### DISCUSSION TITLE
Re : Raman shift plot

### DISCUSSION THREAD

**Discussion by Santoshkumarkaushik** [nguyen-group/QERaman#7] [General]
Dear Sir
I have calculated raman spectra for 2d tmds.
Sir for plotting purpose which column I have to use since there are 10 column in file.
![Uploading 17060165908377767616075667330388.jpg…]()

**Comment by Santoshkumarkaushik** [comment]
[raman_spectra2.txt](https://github.com/nguyen-group/QERaman/files/14024672/raman_spectra2.txt)
This is output file for elaser 2.33 eV. Plz guide me in this regard.
Thanks.

**Comment by nguyen-group** [comment]
Hi,

It depends on your Raman configuration.
The Raman configuration is listed in the output file as: #Raman shift (cm-1), Raman Spectra (XX,YY,ZZ,s+s+,s-s-,XY,YX,s+s-,s-s+)
For example, XX means that the direction of the incident and scattered lights are the same x direction. 
Since the Raman tensors are also written in the output file raman.out, so you can obtain the Raman spectra for any configuration, which is not listed in raman_spectra1.dat

I suggest you should understand the Raman setup in the experiment that you want to calculate first. Some textbooks in Raman might be helpful.

Best,
Nguyen

**Comment by Santoshkumarkaushik** [comment]
Thanks sir
